man/ctags-incompatibilities.7.rst.in

```diff
@@ -18,10 +18,40 @@ SYNOPSIS
 DESCRIPTION
 -----------
 
-TO BE WRITTEN.
+This page describes major incompatible changes introduced to
+Universal-ctags forked from Exuberant-ctags.
 
 Incompatibilities in command line interface
 -------------------------------------------------------------
 
+``--langmap``
+~~~~~~~~~~~~~~
+
+When applying mappings for a name of given source file,
+Exuberant-ctags tests file name patterns AFTER file extensions
+(**e-map-order**). Universal-ctags does this differently; it tests file
+name patterns BEFORE file extensions (**u-map-order**).
+
+This incompatible change is introduced to deal with the following
+situation:
+
+	* "build.xml" as a source file,
+	* The "Ant" parser declares it handles a file name pattern "build.xml", and
+	* The "XML" parser declares it handles a file extension "xml".
+
+Which parser should be used for parsing "build.xml"?  The assumption
+of Universal-ctags is the user may want to use the "Ant" parser; the
+fie name pattern it declares is more specific than the file extension
```

Comment:
Yes. I fixed this in  	0f6569a.
I have to say cspell target  is far from perfect.